iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -384,6 +407,50 @@ public Builder setOnConfigurationListener(
             return this;
         }
 
+        /**
+         * Add a listener to receive configuration changes sent to the device from Cloud IoT
+         * Core.
+         *
+         * <p>Cloud IoT Core resends device configuration every time the device connects to
+         * Cloud IoT Core, so clients should expect to receive the same configuration
+         * multiple times.
+         *
+         * <p>This parameter is optional.
+         *
+         * @param executor the thread the callback should be executed on
+         * @param listener the listener to add
+         * @return this builder
+         */
+        public Builder setOnCommandListener(
+                @NonNull Executor executor, @NonNull OnCommandListener listener) {
+            checkNotNull(executor, "Executor for OnCommandListener");
+            checkNotNull(listener, "OnCommand listener");
+
+            mOnCommandExecutor = executor;
+            mOnCommandListener = listener;
+            return this;
+        }
+
+        /**
+         * Add a listener to receive configuration changes sent to the device from Cloud IoT
```

Comment:
This comments should be about command instead of configuration.

Reply:
:+1: